pandas/_libs/groupby.pyx

```diff
@@ -1213,14 +1201,6 @@ def group_rank(float64_t[:, :] out,
                     out[i, 0] = out[i, 0] / grp_sizes[i, 0]
 
 
-group_rank_float64 = group_rank["float64_t"]
-group_rank_float32 = group_rank["float32_t"]
-group_rank_int64 = group_rank["int64_t"]
-group_rank_uint64 = group_rank["uint64_t"]
-# Note: we do not have a group_rank_object because that would require a
-#  not-nogil implementation, see GH#19560
-
-
```

Comment:
Are these just unused?

Reply:
If you look at get_cython_function in groupby.ops, you'll see we do a thing with checking for `libgroupby func_name` and then `libgroupby.func_name_dtype`.  This pattern was put in place back before we started using fused types for libgroupby, and now cython effectively choose the correct func_name_dtype for us when we just use func_name